docker-compose.yaml

```diff
@@ -0,0 +1,6 @@
+version: '3'
+services:
+    elasticsearch:
+        image: "elasticsearch:6.8.11"
+        ports:
+            - "9200:9200"
```

Comment:
Don't you need environment variables to set user and password ?

Reply:
I don't need it to connect to the db, I can connect without user or password

Reply:
It's better if you can set some user/password